README.md

```diff
@@ -40,131 +40,152 @@ FIEDLER, Niklas, et al. [ImageTagger: An Open Source Online Platform for Collabo
-+ The ALLOWED\_HOSTS
-+ The database settings
-+ The UPLOAD\_FS\_GROUP to the id of the group that should access and create the uploaded images
-
-For the database, postgresql is used. Install it by running `sudo apt install postgresql` on Debian based operating systems. A default database cluster is automatically initialized.
-
-Other systems may require different commands to install the package and the database cluster may
-have to be initialized manually (e.g. using `sudo -iu postgres initdb --locale en_US.UTF-8 -D '/var/lib/postgres/data'`).
-
-To start the postgresql server, run `sudo systemctl start postgresql.service`. If the server should always be started on boot, run `sudo systemctl enable postgresql.service`.
-
-Then, create the user and the database by running
-
-`sudo -iu postgres psql`
-
-and then, in the postgres environment
-
-```
-CREATE USER imagetagger PASSWORD 'imagetagger';
-CREATE DATABASE imagetagger WITH OWNER imagetagger ENCODING UTF8;
-```
-
-where of course the password and the user should be adapted to the ones specified in the database settings in the settings.py.
-
-To initialize the database, run `./manage.py migrate`
-
-To create an administrator user, run `./manage.py createsuperuser`.
-
-`./manage.py runserver` starts the server with the configuration given in the settings.py file.
-
-To create annotation types, log into the application and click on Administration at the very bottom of the home page.
-
-For **production** systems it is necessary to run the following commands after each upgrade
-
-```bash
+## Installing and running Imagetagger
+Imagetagger can be installed and run locally (best for development), in a docker container or in Kubernetes 
+(used in our deployment).
+
+### Locally
+
+1.  #### Install the latest release
+    
+    You should probably do this in a [virtual environment](https://virtualenv.pypa.io/en/stable/) 
+    ```shell
+    git checkout v…
+    pip3 install -r imagetagger/requirements.txt
+    ```
+   
+2.  #### Setup a database server
+    
+    As a database server [postgresql](https://www.postgresql.org/) is required.
+    Please seek a guide specific to your operating system on how to install a server and get it running.
+   
+    Once postgresql is installed, a user and database need to be set up for imagetagger.
+    Of course, the user and password can be changed to something else.
+    ```postgresql
+    CREATE USER imagetagger PASSWORD 'imagetagger';
+    CREATE DATABASE imagetagger WITH OWNER imagetagger;
+    ```
+
+3.  #### Configuring ImageTagger to connect to the database
+    
+    Please see the lower section about application configuration on how to configure ImageTagger for your specific
+    database credentials.
+   
+4.  #### Initialize the database
+    
+    Run the following to create and setup all necessary database tables:
+    ```shell
+    ./manage.py migrate
+    ```
+
+5.  #### Create a user
+    
+    ```shell
+    export DJANGO_CONFIGURATION=Prod
```

Comment:
Here and in 6, `DJANGO_CONFIGURATION` should rather not be set (= set to `Dev`), shouldn't it?

Reply:
No I don't think so but I have clarified what this does in 696b89fcedc12ce9b3a588363b8c112b9dcd98bf